nixos/modules/services/misc/plex.nix

```diff
@@ -30,6 +30,30 @@ in
         default = "plex";
         description = "Group under which Plex runs.";
       };
+
+
+      managePlugins = mkOption {
```

Comment:
I think it's better to remove this option and enable plugins if `extraPlugins == true`.